cl/recap/management/commands/reprocess_recap_dockets.py

```diff
@@ -19,36 +19,30 @@ def extract_unextracted_rds(queue: str) -> None:
     :return: None
     """
 
-    rd_needs_extraction = [
-        x.pk
-        for x in RECAPDocument.objects.filter(
-            (Q(ocr_status=None) | Q(ocr_status=RECAPDocument.OCR_NEEDED))
-            & Q(is_available=True)
-            & ~Q(filepath_local="")
-        ).only("pk")
-    ]
-    count = len(rd_needs_extraction)
-
+    rd_needs_extraction = (
+        RECAPDocument.objects.filter(
+            Q(ocr_status__isnull=True)
+            | Q(ocr_status=RECAPDocument.OCR_NEEDED),
+            is_available=True,
+        )
+        .exclude(filepath_local="")
+        .values_list("pk", flat=True)
+        .order_by()
+    )
+    count = rd_needs_extraction.count()
     # The count to send in a single Celery task
     chunk_size = 100
```

Comment:
100 is a lot to send in a single batch. Maybe 10 would be better?

Reply:
Sure. Added the `--chunk-size` parameter with a default value of 10.